.github/dependabot.yml

```diff
@@ -14,6 +14,22 @@ updates:
     open-pull-requests-limit: 3
     labels:
       - "release-note-none"
+version: 2
+updates:
```

Comment:
Drop these lines, so you only add a new item to the existing `updates`.

Reply:
Done.